group-coordinator/src/main/java/org/apache/kafka/coordinator/group/assignor/PartitionAssignor.java

```diff
@@ -36,8 +36,9 @@ public interface PartitionAssignor {
      * Perform the group assignment given the current members and
      * topic metadata.
      *
-     * @param assignmentSpec The assignment spec.
+     * @param assignmentTopicDescriber The topic and cluster metadata describer {@link AssignmentTopicDescriber}.
+     * @param assignmentSpec The member assignment spec.
      * @return The new assignment for the group.
      */
-    GroupAssignment assign(AssignmentSpec assignmentSpec) throws PartitionAssignorException;
+    GroupAssignment assign(AssignmentTopicDescriber assignmentTopicDescriber, AssignmentSpec assignmentSpec) throws PartitionAssignorException;
```

Comment:
Would it make sense to put `AssignmentTopicDescriber` as the second argument?

Reply:
The order prolly doesn't matter too much but yeah changed it just cause of the names the order you suggested makes sense